### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by Ericson2314** [simonmar/happy#204]
Should we do a release before the package splitting? (The least easy release, hah!)

I thought this was discussed, but I cannot find.

**Comment by int-index** [comment]
I haven’t followed closely, so I don’t know what the difference would be with the last release (1.20.0). If there are features that users are waiting for, then sure, no reason to hold back.

**Comment by Ericson2314** [comment]
I checked, I think the big feature is the bootstrapping. I think the dirt-simple single-package no-bootstrap-happy needed position we're at might be worth releasing before we trade one sort of complexity for another.

I think we were holding off because you were thinking of replacing your tuned custom parser combinators with the slower ones in `base` for brevity, but I don't mind not doing that until later. (Insofar that I don't believe the standard library should have an arbitrary take on parser combinators, this is more "portable" anyways.)

**Comment by int-index** [comment]
Yep, it’s a trade-off between familiarity/perf. I’m not in a hurry to get rid of the ones I’ve implemented.

**Comment by Ericson2314** [comment]
I merged the read me change and uploaded a release candidate https://hackage.haskell.org/package/happy-1.21.0/candidate
